nowcasting_dataset/config/model.py

```diff
@@ -279,6 +301,35 @@ def set_forecast_and_history_minutes(cls, values):
 
         return values
 
+    @root_validator
+    def set_start_and_end_datetime(cls, values):
+        """
+        Set start and end date, if needed.
+
+        Run through the PV and GSP data sources and  if the start or end date are not set,
+        then set them to this values
+        """
+        # It would be much better to use nowcasting_dataset.data_sources.ALL_DATA_SOURCE_NAMES,
```

Comment:
might this function also be a good place to check that `end_datetime > start_datetime`?